rfcs/rfc-009-staking.md

```diff
@@ -0,0 +1,59 @@
+Initial proof-of-stake (i.e. using in-protocol asset, SOL, to provide secure consensus) design ideas outlined here. Solana will implement a proof of stake reward/security scheme for node validators on the network. The purpose is threefold:
```

Comment:
Inconsistency: "proof-of-stake" in first sentence. "proof of stake" in the second. I'd prefer "Proof of Stake" in both or define-then-use an acronym, "Proof of Stake (PoS) ... PoS".